bindings/rust/evmc-declare-tests/src/lib.rs

```diff
@@ -0,0 +1,20 @@
+use evmc_vm::EvmcVm;
+use evmc_vm::ExecutionContext;
+use evmc_vm::ExecutionResult;
+#[macro_use]
+use evmc_declare::evmc_declare_vm;
+
+#[evmc_declare_vm("Foo VM", "ewasm")]
```

Comment:
Since this is a comprehensive test, I'd say we need to VMs declared here:
1) supports single capability and takes version from crate
2) supports multiple capabilities and takes version from declare macro